Added requirements.txt

PJ-Finlay opened the pull request.

PJ-Finlay committed:
Added requirements.txt

PJ-Finlay commented:
Not sure if you want this but I have a working requirements.txt for people that want to use pip.

PJ-Finlay committed:
Changed default to maintaining current image size

PJ-Finlay committed:
Added script to automate style transfer

PJ-Finlay committed:
Fixed bug in previous commit

PJ-Finlay committed:
Changed default height to 720

gordicaleksa commented:
Hey @PJ-Finlay! I don't currently have the bandwidth to deal with NST PRs/issues.

I just dealt with my [DeepDream repo](https://github.com/gordicaleksa/pytorch-deepdream) - did lots of refactoring, added full support for cmd line, some new features, and a Jupyter notebook. And only then I reviewed the PRs/issues.

I'll do the same thing for this repo in 2 months there are a lot of things left on my backlog. Feel free to contribute and if your changes are orthogonal to mine I'll review them and integrate them. Just make sure you follow the coding style of this repository. Thank you! And apologies, I do these in my free time and so as you can see I review my old projects every 2 months.

PJ-Finlay commented:
No worries, thanks!

All I added was:
- a requirements.txt
- a script to automate multiple NSTs
- Changed the default height to 720 (this was something I wanted but might not make sense in the main repo)

Also check out the [neural style transfer video](https://youtu.be/xvawoZMIcSE) I made with this library!
